backend/src/main/kotlin/org/loculus/backend/service/groupmanagement/GroupManagementDatabaseService.kt

```diff
@@ -172,4 +152,31 @@ class GroupManagementDatabaseService(
                 contactEmail = it.contactEmail,
             )
         }
+
+    private fun GroupEntity.updateWith(group: NewGroup) {
```

Comment:
This method looks like one can only update all values at once. Is that what we want? Or do we want to offer users the possibility to just update a single field (and not writing all the other fields to the request, too)?

Reply:
@chaoran-chen Do you wanna weigh in on this?

Reply:
The PUT specification [here](https://httpwg.org/specs/rfc9110.html#PUT) says

> A successful PUT of a given representation would suggest that a subsequent GET on that same target resource will result in an equivalent representation being sent in a [200 (OK)](https://httpwg.org/specs/rfc9110.html#status.200) response.

So I think that it's better if the endpoint user always has to provide the whole object. Otherwise, we also would have to deal with distinguishing whether the user wants to not change something or set it to `null`.